apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -172,6 +172,11 @@ export const GLOBAL_MENU_ITEMS: GlobalMenuItems = [
       label: 'Resources',
       menuItems: [
         [
+          /*           {
```

Comment:
will eventually go here but hidden for now because we don't have many migrated yet